user-guide/Advanced_Modules/Router_Control/Working_with_the_default_RC_config/Selecting_a_matrix.md

```diff
@@ -14,4 +14,4 @@ At the top of the *Router Control* module, underneath the gray header area, you
 
     - two information panels (the left one showing information about the selected input, and the right one showing information about the selected output), and
 
-    - three buttons (Connect, Disconnect, and Lock/Unlock).
+    - three buttons (Connect, Disconnect, and N/A). The "Lock" keyword replaces "N/A" when an input and output is clicked upon in the *Router Control* UI. The "Unlock" option appears when input and output were previously locked.
```

Comment:
use 'selected' instead of 'clicked upon'
This should probably be in a note instead of the full text. Lock/Unlock is the name of the function, but it happens to show N/A when first opening the view of a matrix. This is further explained in the "Locking an output" and "unlocking an output" pages.